backend/db/structure.sql

```diff
@@ -432,6 +432,7 @@ CREATE TABLE public.learning_paths (
     label character varying NOT NULL,
     description character varying NOT NULL,
     badge_id character varying,
+    color character varying NOT NULL,
```

Comment:
this will get auto generated once you run the new migration file, so remove this